### PR TITLE
Added build and dependency instructions to README.

### PR DESCRIPTION
**Pull request by bravehorsie**
Signed-off-by: Roman Grigoriadi <roman.grigoriadi@oracle.com>

### CODE REVIEW

**Comment by bravehorsie** [comment]
Fixes #1298

File: README.md [hunk 1]
```diff
@@ -47,3 +30,200 @@ terms of [OCA](http://oracle.com/technetwork/goto/oca).
 - [METRO project](https://javaee.github.io/metro)
 - [JSR 222](https://jcp.org/en/jsr/detail?id=222)
 - [Download standalone distribution](https://repo1.maven.org/maven2/com/sun/xml/bind/jaxb-ri/2.3.0/jaxb-ri-2.3.0.zip)
+
+## Build instructions
+JavaSE 11 is required to build JAXB.
```
Comment (lukasj) [inline README.md]: also say which maven version is required. Note that if the repo resides in a directory with space in path, then the build fails,... so would be good to either document or fix this ;-)

File: README.md [hunk 2]
```diff
@@ -47,3 +30,200 @@ terms of [OCA](http://oracle.com/technetwork/goto/oca).
...
+
+- **jaxb-ri/external**:
+RNGOM and relaxng-datatype under jaxb-ri/external are tools for working with RelaxNG. It is released separately because it is a dependency of 
+several jaxb-ri projects including jaxb-ri/xsom above.
+
+##Using JAXB with Maven 
+
+###Maven coordinates for JAXB artifacts
+
+-   **java.xml.bind:jaxb-api** API classes for JAXB. Required to compile
```
Comment (lukasj) [inline README.md]: shouldn't this be changed to 'jakarta.xml.bind:jakarta.xml.bind-api'?

File: README.md (hunk 1 above)
Comment (bravehorsie) [reply README.md]: Not sure how does spaces in paths relate to jaxb. Are there any issues regarding that?

Comment (lukasj) [reply README.md]: tests in XJC (?) are failing

Comment (bravehorsie) [reply README.md]: This is fixed in another PR.

File: README.md (hunk 2 above)
Comment (bravehorsie) [reply README.md]: Good catch.

File: README.md (hunk 1 above)
Comment (lukasj) [reply README.md]: the fact that the path XJC is trying to resolve and fails contains '%20'?

Comment (bravehorsie) [reply README.md]: Can you link the issue?

Comment (lukasj) [reply README.md]: there is none yet

Comment (lukasj) [reply README.md]: #1304 not that hard to spot and/or reproduce

Comment (bravehorsie) [reply README.md]: Lets fix that in another PR. JAXB repository doesn't contain such spaces in directory names and so won't affect the build by default.

File: jaxb-ri/HOWTO-build.txt [hunk 3]
```diff
@@ -8,5 +8,10 @@
 # SPDX-License-Identifier: BSD-3-Clause
 #
 
-Three words. "mvn clean install"
+JavaSE 11 and Maven 3.3.9 are required for the build.
```
Comment (lukasj) [inline jaxb-ri/HOWTO-build.txt]: Build jobs are using latest maven - 3.6.0, should they be updated to conform to what is being said here? Or should the text contain 'or newer'?

**Pull request merged**